package.json

```diff
@@ -69,8 +69,8 @@
     "format:css": "npm run format:provided '**/*.scss' '**/*.css'",
     "format:provided": "prettier --write",
     "tube:setup": "./bin/jurassic-tube-setup.sh",
-    "tube:start": "./docker/bin/jt/tunnel.sh",
-    "tube:stop": "./docker/bin/jt/tunnel.sh break",
+    "tube:start": "source ./bin/jurassictube/config.env && jurassictube -u \"$username\" -s \"$subdomain\" -h localhost:8082",
```

Comment:
Hardcoded the port for now because getting it from `docker-compose.yml` was not readable enough to include it in the setup script, but I'm happy to hear other thoughts.